test/test.js

```diff
@@ -1,13 +1,26 @@
+"use strict";
```

Comment:
Is it this error https://travis-ci.org/ewnd9/anki-apkg-export/jobs/179239375#L223 ?